devel/CoreOSMakefiles/Portfile

```diff
@@ -2,6 +2,7 @@ PortSystem              1.0
 
 name                    CoreOSMakefiles
 version                 77
+revision                1
```

Comment:
Don't bump the revision if the code isn't changing.

Reply:
Looks like that's fixed as of 6d86c28.

Reply:
Yes, I did. But I haven't click "resolve conversation" to make things clear and allow Joshua to see that his remarks was addressed.